docs/sources/reference/run.md

```diff
@@ -552,6 +553,22 @@ would be 2*300M, so processes can use 300M swap memory as well.
 We set both memory and swap memory, so the processes in the container can use
 300M memory and 700M swap memory.
 
+By default, processes in a container are killed if an out-of-memory (OOM) occurs.
+To disable this behaviour, use the `--oom-kill-disable` option. Disabling the OOM killer
+should only be done on containers that have the `-m/--memory` option set.
+
+Examples:
+
+    $ docker run -ti -m 100M --oom-kill-disable ubuntu:14.04 /bin/bash
+
+This example limits the memory to 100M and disables the OOM killer for this container.
+
+    $ docker run -ti --oom-kill-disable ubuntu:14.04 /bin/bash
+
+This example is dangerous; there is no memory-limit set for the container and the
+OOM-killer is disabled. This may result in the host running out of memory, and system
+processes being killed to free memory.
```

Comment:
If the OOM is disabled, what part of the system is doing this: "system processes being killed to free memory"